composer.json

```diff
@@ -21,7 +21,7 @@
     "require": {
         "php": "^7.2",
         "knplabs/knp-snappy-bundle": "^1.5",
-        "sylius/sylius": "1.3.*",
+        "sylius/sylius": "~1.4.0",
```

Comment:
In other plugins we require `sylius/sylius` in `^1.4` version. Requiring it in `~1.4.0` version makes me wonder what approach is more suitable for us, WDYT?

Reply:
`^1.4` 👌